MediaSession.cpp

```diff
@@ -600,27 +612,41 @@ CDMi_RESULT MediaKeySession::Decrypt(
        MEMCPY(&ctrContext.qwInitializationVector, f_pbIV, f_cbIV);
     }
 
+/* PRv3.3 support */
 #ifdef PR_3_3
-    if ( NULL == f_pdwSubSampleMapping )
+   if ( NULL == f_pdwSubSampleMapping )
     {
         rgdwMappings[0] = 0;
         rgdwMappings[1] = payloadDataSize;
         f_pdwSubSampleMapping = reinterpret_cast<const uint32_t*>(rgdwMappings);
         f_cdwSubSampleMapping = NO_OF(rgdwMappings);
     }
 
+    if( f_pcbOpaqueClearContent != NULL )
+    {
+        *f_pcbOpaqueClearContent = 0;
+    }
+    if( f_ppbOpaqueClearContent != NULL )
+    {
+        *f_ppbOpaqueClearContent = NULL;
+    }
+
     ChkDR(Drm_Reader_DecryptOpaque(
+#ifdef NETFLIX
         &m_oDecryptContext,
+#else
+        m_oDecryptContext,
+#endif
```

Comment:
Same as above

Reply:
In PR 3.3, Drm_Reader_DecryptOpaque() has decrypt context parameter in double pointer. As this is already under PR_3_3, separated with NETFLIX macro.

Reply:
Same as above.